### ISSUE TITLE
MikesTweaks additional inventory slots not working when LethalThings is enabled

### ISSUE THREAD

**Issue by MrPlaner** [EvaisaDev/LethalThings#16]
Hey there!

Been playing modded LC while having those two mods enabled.
But recently after waiting for mods to update, when me and friends came back, additional slots from MikesTweaks were not working.

The difficult part is... as long as LethalThings is disabled, the additional inventory slots from MTweaks are working fine.
As soon as LethalThings mod is enabled, the only part from MikesTweaks that is not working is the additional slots.

If it's not LethalThings mod fault, please let me know and I will head over to MikesTweaks github!

TL;DR: Then MTweaks and LThings are both on, MT's additional slots are not working.

But if anything I can easily reproduce the issue by installing MikesTweaks and LethalThings.
Then when you run the modded client with just MikesTweaks enabled the default 2 additional slots are present, but when both are enabled the slots return to vanilla 4.

**Comment by EvaisaDev** [comment]
Sounds like a mtweaks problem and not a lethal things problem, i rewrote the utility belt code recently for better support with mods like Reserved Item Slots which may be related. I recommend creating an issue there and mentioning this issue.

**Comment by MrPlaner** [comment]
Actually, for some reason the update literally less than 2 hours ago for... Lategame Upgrades fixed the issue.
I have NO clue why or what caused the issue in the first place.

If comes up again will head over to MTweaks Github then, thanks a lot!
